### DISCUSSION TITLE
alerts on low disk space

### DISCUSSION THREAD

**Discussion by jmls** [amerkurev/doku#33] [General]
is there a capability of sending alerts when the disk space starts running low ?

**Comment by c00ldude1oo** [comment]
Not with doku I believe. You will have to find something else to do so.

**Comment by amerkurev** [comment]
Hi @jmls 

I’ve thought about this too, but I’m not sure if it’s really Doku’s job. To implement this, I’d need to add various notification delivery methods, like what [Gatus](https://github.com/TwiN/gatus) does. Maybe it makes more sense to just add an API endpoint in Doku? That way, something like Gatus could call it and handle the notifications instead.
